File: app/src/main/java/com/neome/api/ent/entAside/sig/SigSpreadsheetRowsGet.java
// neome.ai API - do not change
//

package com.neome.api.ent.entAside.sig;

import com.neome.api.meta.base.Types.MetaIdForm;
import com.neome.api.meta.base.Types.RowId;
import com.neome.api.nucleus.base.sig.Sig;

import org.jetbrains.annotations.Nullable;

import java.util.Map;
import java.util.Set;

@SuppressWarnings("unused")
public class SigSpreadsheetRowsGet extends Sig
{
  @Nullable
  public Map<String, Set<RowId>> dateRowIdSetMap;

  @Nullable
  public Map<String, Set<RowId>> groupByRowIdSetMap;

  public MetaIdForm outputFormId;

  @Nullable
  public RowId[] rowIdSet;
}

File: app/src/main/java/com/neome/api/ent/entAside/sig/SigSpreadsheetRowsGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entAside.sig

import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.RowId
import com.neome.api.nucleus.base.sig.Sig

interface SigSpreadsheetRowsGet : Sig
{
  val dateRowIdSetMap: Map<String, Set<RowId>>?
  val groupByRowIdSetMap: Map<String, Set<RowId>>?
  val outputFormId: MetaIdForm
  val rowIdSet: List<RowId>?
}